beacon_node/network/src/sync/network_context.rs

```diff
@@ -142,13 +151,41 @@ impl<T: BeaconChainTypes> SyncNetworkContext<T> {
             request_id: 1,
             blocks_by_root_requests: <_>::default(),
             blobs_by_root_requests: <_>::default(),
+            data_columns_by_root_requests: <_>::default(),
             range_blocks_and_blobs_requests: FnvHashMap::default(),
             network_beacon_processor,
             chain,
             log,
         }
     }
 
+    // TODO(das): epoch argument left here in case custody rotation is implemented
+    pub fn get_custodial_peers(&self, _epoch: Epoch, column_index: ColumnIndex) -> Vec<PeerId> {
+        let mut peer_ids = vec![];
+
+        for (peer_id, peer_info) in self.network_globals().peers.read().peers() {
```

Comment:
`peers` returns all known peers in the `PeerDB`. Maybe `connected_peers` should be used here instead?